docs/source/knowledgebase/quasi_dags.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>The ability to discount or ignore U2 is crucial. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/517/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
It's a case of conditional ignorability....